### ISSUE TITLE
Config.unobserve is deprecated.

### ISSUE THREAD

**Issue by stramel** [AtomLinter/linter-jscs#40]
Config::unobserve no longer does anything. Call `.dispose()` on the object returned by Config::observe instead.
```
Config.unobserve (/usr/local/share/atom/resources/app/src/config.js:374:19)
LinterJscs.destroy (/home/stramel/.atom/packages/linter-jscs/lib/linter-jscs.coffee:77:18)
```

**Atom:** 0.189.0-edd46e4

**Comment by iam4x** [comment]
Can you do a PR @stramel ? I've just started a new job, I have less time for this :(

**Comment by stramel** [comment]
@iam4x Seems pretty straight forward but I don't have much time currently either. Working 2 jobs doesn't leave much time.

**Comment by iam4x** [comment]
Fixed with #44
